examples/afl_network_proxy/afl-network-server.c

```diff
@@ -358,8 +358,8 @@ int recv_testcase(int s, void **buf) {
 
   if ((size & 0xff000000) != 0xff000000) {
 
-    *buf = afl_realloc((void **)&buf, size);
-    if (unlikely(!buf)) { PFATAL("Alloc"); }
+    *buf = afl_realloc(buf, size);
```

Comment:
Is this correct?

Reply:
what do you mean?

Reply:
@domenukk can you explain what you mean? :)